src/ff/galois_field.rs

```diff
@@ -201,6 +235,7 @@ macro_rules! bit_array_impl {
                 }
             }
 
+            /*
```

Comment:
Note: We probably want this back.

Reply:
And maybe make an equivalent thing for the prime field.

Reply:
With From implemented, the TryFrom implementation isn't necessary.  We should choose (I favour the fallible TryFrom given that we need to handle overflow in most cases).

Reply:
We need it for the generation from PRSS. There, we want to just go modulo the prime for PrimeField, and just take the lower 32 bits for Galois field. We were thinking that *most* cases can use the try_from approach, but at least these should truncate.